pkg/helpers/sqlrole.go

```diff
@@ -0,0 +1,33 @@
+package helpers
+
+type SQLRoleAction string
+
+const (
+	RoleActionDelete = "delete"
+	RoleActionCreate = "create"
+)
+
+type SQLRoleAndAction struct {
```

Comment:
```suggestion
type SQLRoleAction struct {
```

Reply:
The enum was already called that

Reply:
This goes away with the change below though, so resolving this.